optaplanner-core/src/test/java/org/optaplanner/core/impl/solver/recaller/BestSolutionRecallerTest.java

```diff
@@ -87,17 +88,22 @@ public void unimprovedInitializedProcessWorkingSolutionDuringStep() {
 
     @Test
     public void improvedUninitializedProcessWorkingSolutionDurintStep() {
+        DefaultSolverScope solverScope = new DefaultSolverScope();
+        InnerScoreDirector scoreDirector = mock(InnerScoreDirector.class);
+        SolutionDescriptor solutionDescriptor = mock(SolutionDescriptor.class);
+        when(scoreDirector.getSolutionDescriptor()).thenReturn(solutionDescriptor);
+        solverScope.setScoreDirector(scoreDirector);
```

Comment:
This boilerplate code should be extracted into or simply using
  org.optaplanner.core.impl.testdata.util.PlannerTestUtils
For example, that already has PlannerTestUtils.mockScoreDirector()